Makefile.Common

```diff
@@ -155,5 +155,5 @@ moddownload:
 
 .PHONY: updatedep
 updatedep:
-	$(PWD)/internal/buildscripts/update-dep
+	TOPDIR=$(PWD) $(PWD)/internal/buildscripts/update-dep
```

Comment:
Do we necessarily need to introduce TOPDIR if PWD is set to the root already?

Reply:
The usage here is confusing, because it implies that TOPDIR changes throughout the execution but it doesn't, it remains the root of the repo. Whereas `PWD` inside the script will change every time it's called

Reply:
I'll follow up this PR to use SRC_ROOT

Reply:
@dmitryax  PTAL https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17087